### PR TITLE
Add strings for Finland/Sweden Premium news

### PR DESCRIPTION
**Pull request by Vinnl**
This is to announce Premium availability in Sweden and Finland.

image

### CODE REVIEW

**Comment by Vinnl** [comment]
@flodolo The lint errors all appear to be instances of `{ -brand-name-firefox-account(capitalization: "uppercase") }`, so I think those are false positives from the linter.

File: en/app.ftl
Comment (flodolo) [inline en/app.ftl:134]: Could we avoid duplicating content for this? It should be only 4 strings

```
whatsnew-feature-premium-sweden-heading = { -brand-name-relay-premium } available in Sweden
whatsnew-feature-premium-finland-heading = { -brand-name-relay-premium } available in Finland
whatsnew-feature-premium-snippet = Upgrade now and get even more protection…
whatsnew-feature-premium-description = Upgrade now and get even more protection — create unlimited email masks, get a custom email subdomain, and more!
```

It's also likely that we'll need this again in the future.

File: en/app.ftl
Comment (flodolo) [inline en/app.ftl:135]:
```suggestion
whatsnew-feature-premium-sweden-description = Upgrade now and get even more protection — create unlimited email masks, get a custom email subdomain, and more!
```

I believe there should be a space before and after the dash
https://mana.mozilla.org/wiki/display/FIREFOX/Punctuation

**Review by flodolo — CHANGES_REQUESTED** [review]
I need to look into the linter. I had to refactor the code for quotes, and didn't account that now parameterized terms would be in the way :-(

File: en/app.ftl
Comment (Vinnl) [reply en/app.ftl]: Ah good point: a2b02ea

File: en/app.ftl
Comment (Vinnl) [reply en/app.ftl]: Also fixed in a2b02ea, and looped back to content to avoid in the future.

File: en/app.ftl
Comment (flodolo) [inline en/app.ftl:132]: This comment doesn't make a lot of sense anymore?

File: en/app.ftl
Comment (flodolo) [inline en/app.ftl:133]: This is unlikely to happen, and we shouldn't ask locales to do it…

With that said: does this apply to the first string, or both?

Comment (Vinnl) [reply en/app.ftl:133]: I don't know of a good way to do this programmatically, since string length doesn't necessarily correspond to the number of characters, and I think there isn't even a definition of character that represents e.g. Latin letters but doesn't risk splitting glyphs spanning multiple Unicode code points (or something like that - I don't know too much of it) into two.

So it applies to the first string: `-snippet` is an excerpt of `-description`.

Comment (flodolo) [reply en/app.ftl:133]: Oh, I see now. That's something only Relay does, and I would strongly suggest to stop doing that. 

It's based on the assumption that other locales can create a meaningful sentence by cropping the full description at the start :-( 

It would make more sense to crop the translation via CSS, or even better, always design a system that allows to display the full sentence.

Comment (Vinnl) [reply en/app.ftl:133]: I'll bring it up with UX to see if they can come up with a design that avoids the use of snippets.

Unfortunately, I don't think CSS has the ability to truncate something at the two-line cut-off. That said, I don't think this needs toe be a meaningful sentence (not all English strings are either)? It's essentially a teaser showing that you can click through to get the full message, e.g. from this:

image

to this:

image

**Review by flodolo — APPROVED** [review]
Note: let's keep this on hold before merge for a few days. Pontoon is having a few load issues today, no point in exposing it.

**Pull request merged**
